package.json

```diff
@@ -83,7 +83,7 @@
 		"test:debug": "DEBUG=pw:browser* playwright test",
 		"start": "electron .",
 		"start:debug": "ELECTRON_ENABLE_LOGGING=1 electron .",
-		"icon": "del-cli assets/generated && electron-icon-builder --input=assets/youtube-music.png --output=assets/generated",
+		"icon": "del-cli assets/generated && electron-icon-generator --input=assets/youtube-music.png --output=assets/generated -s",
```

Comment:
`electron-icon-generator` does not seem to work, because it calls `./node_modules/@ffflorian/electron-icon-generator/index.js` which does nothing - calling `./node_modules/@ffflorian/electron-icon-generator/cli.js` work OTOH. It might be worth doing a fix upstream or change the call here

Reply:
hmmm why do we even need it?

if there's a good reason can't we just do it ourselves?